src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistory/config_de.properties

```
@@ -11,5 +11,5 @@ Always=Immer
 Only\ for\ users\ with\ configuration\ permission=Nur f�r Benutzer mit Konfigurationsberechtigung anzeigen
 Only\ for\ administrators=Nur f�r Administratoren
 Exclude\ users=Benutzer ausschlie�en
-Show\ change\ message\ window=�nderungsmitteilungs-Fenster anzeigen
-Show\ the\ change\ message\ window\ in\ jobs\'\ configure\ pages.=�nderungsmitteilungs-Fenster in Job-Konfiguration anzeigen
+Show\ change\ message\ dialog=�nderungsnachricht-Dialog anzeigen
+Change message is mandatory=Anderungsnachricht ist zwingend erforderlich
```

```suggestion
Change\ message\ is\ mandatory=Anderungsnachricht ist zwingend erforderlich
```
You need to escape the whitespace, otherwise the string is not found